### DISCUSSION TITLE
v2/md-enhance/zh/guide/presentation/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#321] [Announcements]
# v2/md-enhance/zh/guide/presentation/

VuePress 的 Markdown 增强插件

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/presentation/

**Comment by Lin-Yuhao** [comment]
这个演示没有看懂，只看懂---和--分隔幻灯片，希望能有示例代码，感谢开发者的制作。

**Reply by Mister-Hope** [reply]
源代码在 https://github.com/vuepress-theme-hope/vuepress-theme-hope/blame/main/docs/md-enhance/src/zh/guide/presentation/demo.md
